src/Graph.ts

```diff
@@ -563,6 +468,57 @@ export default class Graph {
 	}
 
 	private fetchAllDependencies (module: Module) {
```

Comment:
This method has gotten REALLY long and difficult to read. Do you think it might be possible to extract some aptly named private methods to improve this? Especially the really long arrow function inside the .map hits my eye here.

Reply:
I've separated out the AST code, and cut down on the function size a bit.

Reply:
👍